rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -157,39 +158,36 @@ def build(self):
             exit(0)
         return self
 
-    def rocalRun(self):
+    def rocal_run(self):
         """ Run the pipeline using rocalRun call
         """
         status = b.rocalRun(self._handle)
-        if(status != types.OK):
-            print("Rocal Run failed")
         return status
 
     def define_graph(self):
         """This function is defined by the user to construct the
         graph of operations for their pipeline.
         It returns a list of outputs created by calling ROCAL Operators."""
-        print("definegraph is deprecated")
+        print("define_graph is deprecated")
         raise NotImplementedError
 
     def get_handle(self):
         return self._handle
 
-    def GetOneHotEncodedLabels(self, array, device):
-        if device=="cpu":
-            if (isinstance(array,np.ndarray)):
-                b.getOneHotEncodedLabels(self._handle, array.ctypes.data_as(ctypes.c_void_p), self._numOfClasses, 0)
-            else: #torch tensor
-                return b.getOneHotEncodedLabels(self._handle, ctypes.c_void_p(array.data_ptr()), self._numOfClasses, 0)
+    def get_one_hot_encoded_labels(self, array, device):
+        if device == "cpu":
+            if (isinstance(array, np.ndarray)):
+                b.getOneHotEncodedLabels(self._handle, array.ctypes.data_as(ctypes.c_void_p), self._num_classes, 0)
+            elif (isinstance(array, torch.Tensor)):
+                return b.getOneHotEncodedLabels(self._handle, ctypes.c_void_p(array.data_ptr()), self._num_classes, 0)
         else:
-            if (isinstance(array,cp.ndarray)):
-                b.getCupyOneHotEncodedLabels(self._handle, array.data.ptr, self._numOfClasses, 1)
-            else: #torch tensor
-                return b.getOneHotEncodedLabels(self._handle, ctypes.c_void_p(array.data_ptr()), self._numOfClasses, 1)
+            if (isinstance(array, cp.ndarray)):
+                b.getCupyOneHotEncodedLabels(self._handle, array.data.ptr, self._num_classes, 1)
+            elif (isinstance(array, torch.Tensor)):
```

Comment:
@SundarRajan28 same comment as above

Reply:
Removed the torch import and usage of torch APIs in the latest commit